server/database/data/client.js

```diff
@@ -0,0 +1,121 @@
+const Client = require('../models/client');
+
+module.exports = async () => {
+  const client = [
+    {
+      fullName: 'Asmaa A.H',
+      email: 'asma@gmail.com',
+      password: '$2b$10$gB8D2bI6/X7xTwsSRs5JQe2BvwOLEdMag8dinhfIY4Eu0tIEWVTr2',
+      avatar:
+        'https://cdn.shopify.com/s/files/1/0074/5432/6862/files/mongoose-envoy-0063-tested_grande.jpg?v=1585927855',
+      mobileNumber: '4555787946664',
+      bankAccounts: [
+        {
+          bankName: 'Quads Bank',
+          accountNumber: 56489413246,
+          branchName: 'alshamal',
+          branchNumber: 56789,
+        },
+        {
+          bankName: 'Unit Bank',
+          accountNumber: 56489413246,
+          branchName: 'albania',
+          branchNumber: 56789,
+        },
+      ],
+      balance: [
+        { type: 'USD', total: 5000 },
+        { type: 'ILS', total: 300 },
+      ],
+      mainBank: 15642313,
```

Comment:
also add `joinDate` for each client, cause there is no default value for that

Reply:
I guess when I add all client, mongoose add default date without I add it because I add default value in the schema and it has `date.now`
@Asmaathabet